Common/Types/test/valuepack.cxx

```diff
@@ -23,9 +23,34 @@
 #include <bitset>
 #include "../include/Types/ValuePack.h"
 
+using TwoFourNinePack = o2::types::ValuePack<unsigned int, 2, 4, 9>;
+
+union Foo {
+  unsigned int value;
+private:
+  struct {
+    unsigned int x:2;
+    unsigned int y:4;
+    unsigned int z:9;
+  };
+public:
+  unsigned int getX() const { return x; }
+  unsigned int getY() const { return y; }
+  unsigned int getZ() const { return z; }
+};
+
+__attribute__((noinline))
+unsigned int Get2(Foo const &f) {
+  return f.getY();
+}
+
+__attribute__((noinline))
+unsigned int Get2(TwoFourNinePack const &f) {
+  return f.get<1,unsigned int>();
+}
```

Comment:
a test making sure that the assembly code for ValuePack is optimized. I am getting the same code as the function using the union.